### PR TITLE
Update to Turf `v2.6.0-beta.1`.

### PR DESCRIPTION
**Pull request by MaximAlien**
Xcode 14 is unable to lint Mapbox Directions when running:
```
$ pod spec lint MapboxDirections-pre.podspec --verbose
```
because of incorrect deployment target in `Turf`.

Update to Turf `v2.6.0-beta.1` aims to resolve this issue.

Cocoapods linting with Xcode 13.2.1 works fine.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,7 @@
 ### Packaging
 
 * This library now requires a minimum deployment target of iOS 12.0 or above, macOS 10.14.0 or above, tvOS 12.0 or above, or watchOS 5.0 or above. Older operating system versions are no longer supported. ([#736](https://github.com/mapbox/mapbox-directions-swift/pull/736))
+* MapboxDirections now requires [Turf v2.6.0-beta.1](https://github.com/mapbox/turf-swift/releases/tag/v2.6.0-beta.1). ([#757](https://github.com/mapbox/mapbox-directions-swift/pull/757))
```
Comment (MaximAlien) [inline CHANGELOG.md:8]: After upgrade to Turf v2.6.0-beta.1 that contains such [changes](https://github.com/mapbox/turf-swift/compare/v2.5.0...v2.6.0-beta.1). Carthage build job that uses Xcode 14 started failing (e.g. https://app.circleci.com/pipelines/github/mapbox/mapbox-directions-swift/1805/workflows/6386f69a-33db-479b-924d-29858d2d09e7/jobs/8067). 

Even though error is not really descriptive I do not think that issue is caused by increasing minimum deployment targets. @OdNairy, would you be able to comment on this one? I think that issue is most likely caused by https://github.com/mapbox/turf-swift/pull/197

Comment (OdNairy) [reply CHANGELOG.md:8]: There are outdated schemes like `Turf TV`. Can you remove them, please?
image

**Comment by kried** [comment]
@MaximAlien I guess this pull request might be closed since the version is already updated to 2.6.1 in #761, right?

**Comment by MaximAlien** [comment]
> @MaximAlien I guess this pull request might be closed since the version is already updated to 2.6.1 in #761, right?

Yep, it can be closed.

**Pull request closed without merging**